web/src/PredictImpactMode.svelte

```diff
@@ -21,8 +35,23 @@
   function pickRoad(f: Feature) {
     $mode = { mode: "impact-detail", road: f };
   }
+
+  async function recalculate(fastSample: boolean) {
+    loading = "Calculating impact";
+    // Render the loading screen before starting the calculation. Unsure why Svelte tick() or one frame doesn't work.
+    await new Promise((resolve) => {
```

Comment:
This is consistently working everywhere, thank you for figuring it out! I dunno why one `tick()` doesn't do the trick.